VideoLocker/res/layout/activity_course_base.xml

```diff
@@ -216,15 +215,18 @@
                     android:layout_width="wrap_content"
                     android:layout_height="wrap_content"
                     android:orientation="vertical"
-                    android:gravity="center_vertical|end|right">
+                    android:gravity="center_vertical|end"
```

Comment:
I've removed the center_vertical from gravity but right gravity is required, otherwise, when the section label below it becomes visible, both are aligned to center.